server.py

```
@@ -107,6 +111,84 @@ def save_config(cfg):
+            return json.load(f)
+    except FileNotFoundError:
+        example = ENVS_EXAMPLE_PATH
+        try:
+            with open(example) as f:
+                cfg = json.load(f)
+            save_envs(cfg)
+            return cfg
+        except FileNotFoundError:
+            pass
+    except Exception:
+        pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)